### PR TITLE
Add RHEL-9.4 10.11 imagestreams

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds new imagestreams for RHEL-9.4

### CODE REVIEW

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
This issue blocks https://github.com/sclorg/ansible-tests/pull/58

File: imagestreams/imagestreams.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@
         app_versions: ["10.3", "10.5"]
 
       - name: CentOS Stream 9
-        app_versions: ["10.5"]
+        app_versions: ["10.5", "10.11"]
```
Comment (zmiklank) [inline imagestreams/imagestreams.yaml:18]: Don't we want to set 10.11-el9 as latest for CentOS Stream too?

Comment (phracek) [reply imagestreams/imagestreams.yaml:18]: Why not?

Comment (pkubatrh) [reply imagestreams/imagestreams.yaml:18]: Yes, I would expect having both c9s and rhel9 share the same latest image version.

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
It seems, like some outage. locally it passing

```
check_latest_tag: Compare tags['name']:'latest' against version:'10.11'
Latest tag found.
Imagestreams contains the latest version.
Already on project "default" on server "https://api.core-serv-ocp.hosted.psi.rdu2.redhat.com:6443".
 [PASSED] for 'openshift-remote-cluster' test_mariadb_integration (00:03:14)
 [PASSED] for 'openshift-remote-cluster' test_mariadb_imagestream (00:01:13)
 [PASSED] for 'openshift-remote-cluster' test_mariadb_template (00:01:12)
 [PASSED] for 'openshift-remote-cluster' run_latest_imagestreams (00:00:00)

OpenShift tests for rhel9/mariadb-1011:1 succeeded.
[root@root-1mt-rhel-8 mariadb-container]#
```

**Pull request merged**